RoosterPlanner.Service/Services/TaskService.cs

```diff
@@ -233,5 +240,102 @@ public async Task<TaskResult<Category>> RemoveCategory(Category category)
 
             return result;
         }
+
+        public async Task<TaskResult<ProjectTask>> RemoveProjectTask(ProjectTask projectTask)
+        {
+            if (projectTask == null)
+                throw new ArgumentNullException("projectTask");
```

Comment:
Tegenwoordig wordt vaak nameof(projectTask)  gebruikt (waarbij 'projectTask' de parameter is.